design/ceph-volume-provisioning.md

```diff
@@ -51,7 +51,7 @@ nodes or devices.
   storage:
     config:
       # whether to encrypt the contents of the OSD with dmcrypt
-      encryptDevice: true
+      encryptedDevice: "true"
```

Comment:
Could you also add this setting to these two locations? (probably next to osdsPerDevice)
- cluster.yaml: Either comment it out or set it to false by default
- ceph-cluster-crd.md: Add a description for this new setting in the section for osd configuration settings. We also might want to mention the hang in some environments such as minikube?

Reply:
adding these to the documentation-specific pr